### PR TITLE
Daily classification redesign

### PR DESCRIPTION
**Pull request by alicejo**

### CODE REVIEW

File: app/src/main/res/layout/activity_daily_rating_input.xml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:orientation="horizontal" android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:gravity="top|center_horizontal">
+    <ToggleButton
+        android:drawableTop="@drawable/very_bad_day"
+        android:textOn="@string/very_bad_day"
+        android:textOff="@string/very_bad_day"
+        style="@style/DailyRatingIput"
```
Comment (gewo) [inline app/src/main/res/layout/activity_daily_rating_input.xml]: Input

File: app/src/main/res/layout/content_diary_card_overview.xml [hunk 2]
```diff
@@ -10,7 +10,9 @@
     tools:context="com.bluebird_tech.puffin.activities.DiaryCardOverview"
     tools:showIn="@layout/activity_diary_card_overview">
 
+
     <include layout="@layout/date_switcher" />
     <include layout="@layout/activity_switcher" />
+    <include layout="@layout/activity_daily_rating_input"/>
```
Comment (gewo) [inline app/src/main/res/layout/content_diary_card_overview.xml]: dd

File: app/src/main/res/values-de/strings.xml [hunk 3]
```diff
@@ -119,4 +119,9 @@
     <string name="diary_card_no_content">Diary Card\nnoch nicht ausgefüllt</string>
     <string name="title_activity_emotion_settings">Emotionen anpassen</string>
     <string name="today">Heute</string>
+    <string name="very_bad_day">Furchtbar</string>
```
Comment (gewo) [inline app/src/main/res/values-de/strings.xml]: array?

File: app/src/main/res/values/styles.xml [hunk 4]
```diff
@@ -109,4 +109,15 @@
         <item name="contentPaddingBottom">16dp</item>
     </style>
 
+    <style name="DailyRatingIput" parent="android:Widget.Holo.Button.Borderless">
```
Comment (gewo) [inline app/src/main/res/values/styles.xml]: Input

File: app/src/main/res/layout/content_diary_card_overview.xml (hunk 2 above)
Comment (alicejo) [inline app/src/main/res/layout/content_diary_card_overview.xml]: dd?

**Pull request merged**